ibis/backends/tests/test_aggregation.py

```diff
@@ -94,20 +94,22 @@ def mean_udf(s):
         lambda t: (t.int_col % 3).mode(),
         lambda t: (t.int_col % 3).mode().iloc[0],
         id='mode',
-        marks=pytest.mark.notyet(
-            [
-                "bigquery",
-                "clickhouse",
-                "datafusion",
-                "impala",
-                "mysql",
-                "pyspark",
-                "mssql",
-                "trino",
-                "druid",
-            ],
-            raises=com.OperationNotDefinedError,
-        ),
+        marks=[
+            pytest.mark.notyet(
+                [
+                    "bigquery",
+                    "clickhouse",
+                    "datafusion",
+                    "impala",
+                    "mysql",
+                    "mssql",
+                    "trino",
+                    "druid",
+                ],
+                raises=com.OperationNotDefinedError,
+            ),
+            pytest.mark.notimpl(["pyspark"], raises=AttributeError),
```

Comment:
You've got the compile dispatch for `ops.Mode` in the `pyspark` backend above, but that is saying that the tests aren't hitting it and you've got the `notimpl` here.  That seems odd.  Should we remove the `compile` rule altogether?

Reply:
Hm, yeah. It's implemented in pyspark 3.4, but we're not testing that version yet because it breaks a ton of stuff.

Reply:
Median too, so I'll remove that.

Reply:
Fixed.